tests/SignupApiTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class SignupApiTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /*
+     * Base URL for the Api.
+     */
+    protected $signupsApiUrl = 'api/v2/signups';
+
+    /**
+     * Test that a POST request to /signupss creates a new signup.
+     *
+     * @group creatingAPhoto
+     * @return void
+     */
+    public function testCreatingASignup()
+    {
+    	// Create test signup
+        $signup = [
+            'northstar_id'     => str_random(24),
+            'campaign_id'      => $this->faker->randomNumber(4),
+            'campaign_run_id'  => $this->faker->randomNumber(4),
+            'source'           => 'the-fox-den',
+            'do_not_forward'   => true,
+        ];
+
```

Comment:
@chloealee 1. thank you! I didn't realize there was a Signup one 2. I don't think I was to use that in this case because I want to create the signup via the API